registry.json

```diff
@@ -3871,5 +3871,19 @@
     "telegram": "w_o_u_t",
     "operatorName": "Wout",
     "github": "tiligallery"
+  },
+  {
+    "ticker": "REYTS",
+    "poolId": "b3471fb159f5e8b345018a5c309afd91f06bf99552b8e91eb9980c9d",
+    "social": {
+      "website": "https://www.reyts.com/reytspool/",
+      "twitter": "Reytsapp",
+      "facebook": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "",
```

Comment:
You must have a telegram handle to join the alliance. This must be the telegram of an individual and not a group or channel